features/step_definitions/images_appartient_rubrique.rb

```diff
@@ -0,0 +1,52 @@
+require 'capybara/cucumber'
```

Comment:
This file should not exist (see above). And if it existed it should have an english name.